files/common/var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -116,12 +116,6 @@
     extra_zfs_properties:
       mountpoint: /var/dlpx-update
       compression: gzip
-      #
-      # internal-qa migration images are currently 5G in size and internal-dev
-      # are 11.5G. We need double that space as the uploaded image is unpacked
-      # in the same dataset, thus requiring a minimum of 23G. We use 30G to
-      # leave a safety margin of a few GBs.
-      #
       quota: 30g
```

Comment:
This was originally set to "15g", and bumped to "30g" to support migration.. should we now revert it back to "15g"?

Reply:
I'd rather leave it as is, given that the new upgrade images are significantly larger than on Illumos. 

Also when we reduce the quota, we run the risk of getting a failure if the dataset already uses more space than the quota, which would require some special logic.

Reply:
I understand the risk. I'm still not convinced it's correct to leave the quota artificially inflated, simply because we needed more space in prior releases. The quota is used to limit the number of upgrade images that a customer can upload at any given time, so by leaving it inflated, we regress on that design.

If you don't want to address this right now, can you please open a bug in JIRA so that we can ensure it gets addressed in time for the next release; or at least we can decide as a team that we want to defer this work.

Reply:
For posterity, "15g" was previously used as it's a little more the twice the size of a typical upgrade image for Linux based upgrades:
```
$ du -sh external-standard.upgrade.tar
6.2G    external-standard.upgrade.tar
```
And the original design was for the quota to limit uploads to (at most) two upgrade images at any given time.

Reply:
I've opened https://jira.delphix.com/browse/DLPX-68660